src/api/BoreholeLockService.cs

```diff
@@ -20,6 +20,9 @@ public async Task<bool> IsBoreholeLockedAsync(int? boreholeId, string? subjectId
             .SingleOrDefaultAsync(u => u.SubjectId == subjectId)
             .ConfigureAwait(false) ?? throw new InvalidOperationException($"Current user with subjectId <{subjectId}> does not exist.");
 
+        // Admins can always edit boreholes
+        if (user.IsAdmin) return false;
```

Comment:
Q: Müssten wir hier nicht das Lock auf den Admin übertragen um "lost updates" zu verhindern? Soll das Locken eines Boreholes nicht gerade diese Problematik verhindern?

Reply:
Nein, ein Admin ist davon ausgenommen und soll keine neuen Workflows starten, wenn er selber etwas an einem bestehenden Borehole editiert. _Lost Updates_ gibt es in diesem Sinne nicht, weil das Locking nicht so weit geht: Es beruht lediglich auf einer vordefinierten Zeit und behält keine Updates zurück oder ähnlich. Das ganze Locking bzw. deren Umsetzung aus dem Legacy-API muss sicher nochmals genauer angeschaut werden.